src/ZNCString.cpp

```diff
@@ -1239,7 +1239,18 @@ short CString::ToShort() const {
     return (short int)strtol(this->c_str(), (char**)nullptr, 10);
 }
 unsigned short CString::ToUShort() const {
-    return (unsigned short int)strtoul(this->c_str(), (char**)nullptr, 10);
+    unsigned short int ret;
+    unsigned long int val;
+    val = strtoul(this->c_str(), (char**)nullptr, 10);
+
+    // We need short ints greater than 16bits as much as men need nipples
```

Comment:
be more respectful of the source code

Reply:
Ohh, sorry, it was an internal note to remember where it was, I forgot to remove it. I'll surely remove it!